docker/dist/Dockerfile.dist.centos7

```diff
@@ -16,22 +16,10 @@
 #  under the License.
 
 #syntax=docker/dockerfile:1.2
-FROM centos:centos7
+FROM joshfischer1108/heron-centos7
```

Comment:
This is expected? Official image should be better I feel?

Reply:
I'm up for any suggestions.  We can create a company or organization under Dockerhub, and store the base image there.  Or something else if you prefer.

Reply:
We can create an org like pulsar did https://hub.docker.com/r/apachepulsar/pulsar, but at the end of the day it's still just a dockerhub account.